### DISCUSSION TITLE
Sorting of variables and files in the outline

### DISCUSSION THREAD

**Discussion by JWigh** [codefori/vscode-ibmi#2259] [Ideas]
Is it possible in the renderer of the outline view to add sorting mechnism to get the files in alphabetical order and the variables aswell?

**Comment by worksofliam** [comment]
@JWigh The outline view already supports modes like this. This is something VS Code does, not something we do:

<img width="470" alt="image" src="https://github.com/user-attachments/assets/d6feb7d1-89e6-4c71-a087-5f04ca51e1c4">
